tensorflow/core/kernels/mkl/mkl_einsum_op.cc

```diff
@@ -106,8 +106,9 @@ struct MklEinsumHelper {
 
     // Compute parameters for DNNL matmul primitive.
     MklBatchMatMulHelper bmm;
-    auto params = bmm.CreateMatMulParams(lhs.shape(), rhs.shape(), out_shape,
-                                         trans_x, trans_y);
+    string prefix = "einsum";
+    auto params = bmm.CreateMatMulParams(prefix, lhs.shape(), rhs.shape(),
+                                         out_shape, trans_x, trans_y);
```

Comment:
Nit: Same here. Maybe
```suggestion
    auto params = bmm.CreateMatMulParams("einsum", lhs.shape(), rhs.shape(),
                                         out_shape, trans_x, trans_y);
```
or
```suggestion
    auto params = bmm.CreateMatMulParams(/*prefix=*/"einsum", lhs.shape(), rhs.shape(),
                                         out_shape, trans_x, trans_y);
```
?

Reply:
same reason as above.